aten/src/ATen/native/native_functions.yaml

```diff
@@ -2815,6 +2815,24 @@
 - func: mean.names_out(Tensor self, Dimname[1] dim, bool keepdim=False, *, ScalarType? dtype=None, Tensor(a!) out) -> Tensor(a!)
   device_check: NoCheck   # TensorIterator
 
+- func: nanmean(Tensor self, *, ScalarType? dtype=None) -> Tensor
+  device_check: NoCheck   # TensorIterator
```

Comment:
"#TensorIterator" seems misleading here because nanmean is a composite operation (although it's true that the ops it calls are implemented using TensorIterator). It's probably more accurate to comment this as a "#Composite"